ledger-tool/src/main.rs

```diff
@@ -2471,54 +2477,50 @@ fn main() {
             };
             let genesis_config = open_genesis_config_by(&ledger_path, arg_matches);
             let include_sysvars = arg_matches.is_present("include_sysvars");
-            let exclude_account_data = arg_matches.is_present("exclude_account_data");
             let blockstore = open_blockstore(
                 &ledger_path,
                 AccessType::TryPrimaryThenSecondary,
                 wal_recovery_mode,
             );
-            match load_bank_forks(
+            let (bank_forks, ..) = load_bank_forks(
```

Comment:
It'll look like a lot has changed, but it's mostly whitespace. I recommend ignoring whitespace.